crfs.go

```diff
@@ -946,7 +946,7 @@ func (n *node) Attr(ctx context.Context, a *fuse.Attr) error {
 	a.Uid = uint32(n.te.Uid)
 	a.Gid = uint32(n.te.Gid)
 	a.Rdev = uint32(unix.Mkdev(uint32(n.te.DevMajor), uint32(n.te.DevMinor)))
-	a.Nlink = 1 // TODO: get this from te once hardlinks are more supported
+	a.Nlink = uint32(n.te.NumLink + 1)
```

Comment:
No, not an unconditional plus one. I meant zero should mean one and one should mean one and two should mean two.

In Go, the zero value should always be reasonable. If we really need to represent zero (perhaps in the future), then we can declare that negative values map to zero.